internal/server/stat/golden/get_stat_date_within_place/CA_County.json

```diff
@@ -0,0 +1,163 @@
+{
+  "data": {
+    "Count_Person": {
+      "stat_date": [
+        {
+          "date": {
+            "1980": 56,
+            "1981": 56,
+            "1982": 56,
+            "1983": 56,
+            "1984": 56,
+            "1985": 56,
+            "1986": 56,
+            "1987": 56,
+            "1988": 56,
+            "1989": 56,
+            "1990": 56,
+            "1991": 56,
+            "1992": 56,
+            "1993": 56,
+            "1994": 56,
+            "1995": 56,
+            "1996": 56,
+            "1997": 56,
+            "1998": 56,
+            "1999": 56,
+            "2000": 56,
+            "2001": 56,
+            "2002": 56,
+            "2003": 56,
+            "2004": 56,
+            "2005": 56,
+            "2006": 56,
+            "2007": 56,
+            "2008": 56,
+            "2009": 56,
+            "2010": 56,
+            "2011": 56,
+            "2012": 56,
+            "2013": 56,
+            "2014": 56,
+            "2015": 56,
+            "2016": 56,
+            "2017": 56,
+            "2018": 56,
+            "2019": 43
+          },
+          "metadata": {
+            "import_name": "CensusPEP",
+            "provenance_url": "https://www.census.gov/programs-surveys/popest.html",
+            "measurement_method": "CensusPEPSurvey"
+          }
+        },
+        {
+          "date": {
+            "2011": 58,
+            "2012": 58,
+            "2013": 58,
+            "2014": 58,
+            "2015": 58,
+            "2016": 58,
+            "2017": 58,
+            "2018": 58,
+            "2019": 58
+          },
+          "metadata": {
+            "import_name": "CensusACS5YearSurvey",
+            "provenance_url": "https://www.census.gov/",
+            "measurement_method": "CensusACS5yrSurvey"
+          }
+        },
+        {
+          "date": {
+            "2000": 58,
+            "2010": 58,
+            "2020": 58
+          },
+          "metadata": {
+            "import_name": "USDecennialCensus_RedistrictingRelease",
+            "provenance_url": "https://www.census.gov/programs-surveys/decennial-census/about/rdo/summary-files.html",
+            "measurement_method": "USDecennialCensus"
+          }
+        },
+        {
+          "date": {
+            "2010": 40,
+            "2011": 40,
+            "2012": 40,
+            "2013": 40,
+            "2014": 40,
+            "2015": 40,
+            "2016": 40,
+            "2017": 40,
+            "2018": 40,
+            "2019": 41
+          },
+          "metadata": {
+            "import_name": "CensusACS1YearSurvey",
+            "provenance_url": "https://www.census.gov/",
+            "measurement_method": "CensusACS1yrSurvey"
+          }
+        },
+        {
+          "date": {
+            "2011": 54,
+            "2012": 1,
+            "2014": 2
+          },
+          "metadata": {
+            "import_name": "HumanCuratedStats",
```

Comment:
We changed the name explicitly and decided to show this as "HumanCuratedStats". @pradh On the other hand, do we still need to surf this externally or could just filter this in Flume?

Reply:
I thought we went through it changed the name?  With import-groups we can drop this from all the named import-groups, so it will only show up in laelaps cache.

Reply:
Yes, HumanCuratedStats is the changed name. And dropping it from non-laelaps groups sounds good.